modules/grafana/files/dashboards/platform_health_Q3_19-20_whitehall_errors.json

```diff
@@ -6,7 +6,7 @@
   "gnetId": null,
   "graphTooltip": 0,
   "hideControls": false,
-  "id": null,
```

Comment:
I think the changes in this file shouldn't be here, as they're for the Whitehall dashboard.